### ISSUE TITLE
Ruins context menu in 52.0

### ISSUE THREAD

**Issue by KOLANICH** [takachanz/savelinkmenus#2]

**Comment by Fabiensk** [comment]
**Steps**:
In Firefox 52.0, do a long tap on a link
**Current result**
The text of the link is selected and the "copy to clipboard" bar is displayed instead of the context menu. So it is for example not possible to open a link in a new tab (or to use the add-on features)
**Expected**
Context menu with the add-on items.

**Comment by Evengard** [comment]
Can confirm, disabling this exact addon fixes the problem in FF 52

**Comment by particlese** [comment]
I can confirm that the bug is present with version "0.2.1.1-signed.1-signed" of the plugin and Firefox 52.0 and 52.0.1.  I never noticed the problem prior to Firefox 52.0.

**Partial workaround**
Disable both "Save Selected Links" and "Save Selected Images" in the add-on's options, or use about:config to set the following options to **false**:
* extensions.savelinkmenus.saveselectedimages.enabled
* extensions.savelinkmenus.saveselectedlinks.enabled

(Note:  For me, the four options didn't display in the add-on options interface unless I first disabled the add-on, then enabled it right after.  This happens with all my add-ons, though, so it's not a bug for this one.)

**Comment by maidik** [comment]
Yep, same here. Confirmed. Not sure what FF version this crapped out with, but it was working fully before.

**Comment by halcion** [comment]
So that's it?! Why is it still on the Firefox addons page if it's completely broken and abandonded. WTF

**Comment by lfrankel** [comment]
I don't suppose anyone wants to fork this repo and fix it? :(

**Comment by KOLANICH** [comment]
I guess this repo can be deleted - this ext is not a webext.
